README.md

```diff
@@ -70,7 +70,7 @@ First, uncomment the `console.log` under Deliverable 3 in the `index.js` file to
 Using the `traveler` object, update the DOM to show the traveler's *name*, *nickname*, and *photo* in the appropriate places. For the `<img>` tag, make sure to update the `src` attribute *and* the `alt` attribute (it's important for accessibility).
 
 **YOUR NOTES**
-```
+```You can access traveler object from data.js file in index.js file becaues index.js inherits from data.js? 
```

Comment:
We can access the traveler object from the data.js file because it's a global variable and its content is loaded in from the same HTML file as our index.js file (so they both have access to the same global scope)!